controllers/operands/kubevirt_test.go

```diff
@@ -543,6 +553,8 @@ Version: 1.2.3`)
 			Expect(*mc.ParallelMigrationsPerCluster).To(Equal(parallelMigrationsPerCluster))
 			Expect(*mc.ProgressTimeout).To(Equal(progressTimeout))
 			Expect(*mc.Network).To(Equal(network))
+			Expect(*mc.AllowAutoConverge).To(Equal(allowAutoConverge))
+			Expect(*mc.AllowPostCopy).To(Equal(allowPostCopy))
```

Comment:
allowAutoConverge & allowPostCopy are not changed. Can we drop them?

```suggestion
			Expect(*mc.AllowAutoConverge).To(BeTrue())
			Expect(*mc.AllowPostCopy).To(BeTrue())
```